index.js

```diff
@@ -1,21 +1,26 @@
-
-
-function submitbtn() {
-    document.querySelector('.secondFace').style.display = 'block';
-    var getSelectedValue = document.querySelector(
-        'input[name="score"]:checked');
-
-    if (getSelectedValue != null) {
-        document.getElementById("selected").innerHTML
-            = getSelectedValue.value;
+/**
+ * Shows in DOM the secondFace section, along with the selected value.
+ * @param {HTMLInputElement} selectedValue
+ */
+const renderSecondFade = (selectedValue) => {
+    // Looks for elements in DOM before changing them.
+    const secondFace = document.querySelector('.secondFace');
+    const selectedSpanText = document.querySelector("#selected");
```

Comment:
Since we're using querySelector, don't use getElementById to keep code consistency.